graphbolt/build.sh

```diff
@@ -16,7 +16,19 @@ fi
 # TORCH_CUDA_ARCH_LIST and we need to at least compile for Volta. Until
 # https://github.com/NVIDIA/cccl/issues/1083 is resolved, we need to compile the
 # cuda/extension folder with Volta+ CUDA architectures.
-CMAKE_FLAGS="-DCUDA_TOOLKIT_ROOT_DIR=$CUDA_TOOLKIT_ROOT_DIR -DUSE_CUDA=$USE_CUDA -DGPU_CACHE_BUILD_DIR=$BINDIR -DTORCH_CUDA_ARCH_LIST=Volta"
+TORCH_CUDA_ARCH_LIST="Volta"
+if ! [[ -z "${CUDAARCHS}" ]]; then
+  # The architecture list is passed as an environment variable, we set
+  # TORCH_CUDA_ARCH_LIST to the latest architecture.
+  CUDAARCHSARR=(${CUDAARCHS//;/ })
+  LAST_ARCHITECTURE=${CUDAARCHSARR[-1]}
```

Comment:
`[-1]` is assumed that the passed arches are in ascending order of compute capability? Is this always true?

Reply:
The architectures come from DGL. There, newer architectures are always appended at the end of the list. Overall, CUDA architecture lists are almost always in the old to new sorted order.